Dockerfile

```diff
@@ -18,6 +18,11 @@ RUN dnf install -y \
   iptables \
   fuse-sshfs
 
+# install cosign
+RUN curl -L -O https://github.com/sigstore/cosign/releases/download/v2.0.1/cosign-2.0.1.x86_64.rpm
```

Comment:
nit, nonblocking: can we move these into one `RUN` command to reduce layer count?

Reply:
Good point, updated.